src/net.cpp

```diff
@@ -2813,35 +2813,37 @@ bool CConnman::NodeFullyConnected(const CNode* pnode)
 
 void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
 {
-    size_t nMessageSize = msg.data.size();
-    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n",  SanitizeString(msg.m_type), nMessageSize, pnode->GetId());
+    size_t message_size = msg.data.size();
+    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n", SanitizeString(msg.m_type), message_size, pnode->GetId());
 
     // make sure we use the appropriate network transport format
-    std::vector<unsigned char> serializedHeader;
-    pnode->m_serializer->prepareForTransport(msg, serializedHeader);
-    size_t nTotalSize = nMessageSize + serializedHeader.size();
+    std::vector<unsigned char> serialized_header;
+    pnode->m_serializer->prepareForTransport(msg, serialized_header);
+    size_t total_size = message_size + serialized_header.size();
```

Comment:
d093d261e2df27b2466ffe4e92bd48f22818d326
```suggestion
    const size_t total_size = message_size + serialized_header.size();
```